service/storage_service.cc

```diff
@@ -1221,7 +1191,7 @@ future<> storage_service::on_change(inet_address endpoint, application_state sta
         } else if (move_name == sstring(versioned_value::STATUS_MOVING)) {
             handle_state_moving(endpoint, pieces);
         } else if (move_name == sstring(versioned_value::HIBERNATE)) {
-            co_await handle_state_replacing(endpoint);
+            slogger.error("endpoint={} went into HIBERNATE state, this is no longer supported.  Upgrade the node to the current version.", endpoint);
```

Comment:
It is not clear which should be be upgraded. We can tell the user to use a new version to perform the replace operation.

Reply:
Perhaps a warning is enough, this runs on the existing node, it is not something we could not recover from.

Reply:
Fixed in v2 ([b625726](https://github.com/scylladb/scylladb/commit/b625726d012825c4584737dcbb20ec3a06c6d153))

Reply:
I see the message at line 1194 is not changed in v2. We can make it more explicitly that the user needs to upgrade the replacing node.

Reply:
Please see the latest patch: https://github.com/scylladb/scylladb/pull/12349/commits/c9993f020d2b389d23ab49231960683ca6c709f7
```
            slogger.warn("endpoint={} went into HIBERNATE state, this is no longer supported.  Use a new version to perform the replace operation.", endpoint);
```

Reply:
It is strange the page does not show the latest patch. 

![image](https://user-images.githubusercontent.com/320076/208611130-23090088-e024-43f2-8423-f5b372af25e8.png)

Reply:
why?  This looks like the latest patch to me

Reply:
It is. The screenshot was from the updated version.